consensus/mock/blockProcessorMock.go

```diff
@@ -29,7 +30,7 @@ func (blProcMock *BlockProcessorMock) SetNumProcessedObj(numObj uint64) {
 
 }
 
-func (blProcMock *BlockProcessorMock) ApplyProcessedMiniBlocks(miniBlocks map[string]map[string]struct{}) {
+func (blProcMock *BlockProcessorMock) ApplyProcessedMiniBlocks(miniBlocks *processedMb.ProcessedMiniBlockTracker) {
```

Comment:
The `ProcessedMiniBlockTracker` should implement an interface and pass that around, so when we will add more tests in the block/metablock processors we can mock this and have a simpler setup for unit tests.

Reply:
i will let it as it is because if i will create a interface code will became more complicated